SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -144,11 +152,19 @@ void USpatialClassInfoManager::CreateClassInfoForClass(UClass* Class)
 
 	TArray<UFunction*> RelevantClassFunctions = SpatialGDK::GetClassRPCFunctions(Class);
 
+	// Save AlwaysWrite RPCs to validate there's at most one per class.
```

Comment:
This is a nice check, but we really want to identify any actor where it's combination of actor+components rpcs doesn't more than one one always write rpc. We could track this in the ClassInfoManager for static components, but dynamic components are more problematic. Thoughts on how this check might be safer?

Reply:
An easy way to prevent that now would be to just disallow always write RPCs on subobjects. At the moment, this wouldn't go against the main use case. If we decide to support it in the future, we would likely need to think about how to allow more than one channel, as otherwise the workaround would just be to route these RPCs through the actor.